pom.xml

```diff
@@ -3,7 +3,7 @@
   <modelVersion>4.0.0</modelVersion>
   <groupId>com.amazonaws</groupId>
   <artifactId>aws-java-sdk-pom</artifactId>
-  <version>1.11.762</version>
+  <version>1.11.762-SNAPSHOT</version>
```

Comment:
Set the version to `1.11.762-SNAPSHOT` so the maven release plugin will treat it appropriately.

Reply:
changed to `1.11.762-dw1-SNAPSHOT`